conode/setup-then-start.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/sh
```

Comment:
This was needed by the Dockerfile. So either put it back or figure out if the Dockerfile can go without it. (This is what makes Docker containers create their own private key on boot if there is none.)